src/core/libraries/VestingMath.sol

```diff
@@ -99,19 +99,21 @@ library VestingMath {
     /// @dev Lockup linear model uses the following distribution function:
     ///
     /// $$
-    /// f(x) = x * d + c
+    /// f(x) = x * sa + s + c
     /// $$
     ///
     /// Where:
     ///
-    /// - $x$ is the elapsed time divided by the stream's total duration.
-    /// - $d$ is the deposited amount.
-    /// - $c$ is the cliff amount.
+    /// - $x$ is the elapsed time in the streamable range divided by the streamable total duration.
```

Comment:
Since you have already defined the total duration as the streamable range.

```suggestion
    /// - $x$ is the elapsed time in the streamable range divided by the streamable range.
```